docs/sdk/api/sdkMiddlewareHttp.md

```diff
@@ -24,6 +24,12 @@ Creates a [middleware](/sdk/Glossary.md#middleware) to handle HTTP requests for
 2. `includeResponseHeaders` *(Boolean)*: flag whether to include the response headers in the response, if omitted headers is omitted from response
 3. `includeOriginalRequest` *(Boolean)*: flag whether to include the original request sent in the response. Can be useful if you want to see the final request being sent.
 4. `maskSensitiveHeaderData` *(Boolean)*: flag to mask sensitie data in the header. e.g. Authorization token
+5. `enableRetry` *(Boolean)*: flag to enable retry on network and 500 requests
```

Comment:
Most SDKs have it enabled by default. See Amazon or Paypal SDKs for example. As a browser user i would rather wait a bit longer (2-3 retries) and see successful processing instead of embarrassing 5xx error. For browser you can use shorter and fixed time interval.